src/main/java/com/faforever/client/coop/CoopController.java

```diff
@@ -243,7 +263,10 @@ private void loadLeaderboard() {
         .thenAccept(coopLeaderboardEntries -> {
           AtomicInteger ranking = new AtomicInteger();
           coopLeaderboardEntries.forEach(coopResult -> coopResult.setRanking(ranking.incrementAndGet()));
-          fxApplicationThreadExecutor.execute(() -> leaderboardTable.setItems(observableList(coopLeaderboardEntries)));
+          fxApplicationThreadExecutor.execute(() -> {
+            leaderboardTable.setItems(observableList(coopLeaderboardEntries));
+            leaderboardFilteredTable = new FilteredList<>(observableList(coopLeaderboardEntries));
```

Comment:
Instead of creating the list here we should create the list as a private final field and then add the elements to it.

This would allow us to call the setitems of the leaderboard table only once instead of multiple times.

Reply:
> Instead of creating the list here we should create the list as a private final field and then add the elements to it.
> 
> This would allow us to call the setitems of the leaderboard table only once instead of multiple times.

I tried to use the setAll method on the list but it gives null error. Something like this: 

```leaderboardFilteredList.setAll(observableList(coopLeaderboardEntries));```

I initialized the list as ```private FilteredList<CoopResultBean> leaderboardFilteredList = new FilteredList<>(FXCollections.observableArrayList());```

Reply:
What did it say was null?